torchsparse/nn/modules/conv.py

```diff
@@ -193,3 +203,47 @@ def forward(self, inputs: SparseTensor):
         outputs = outputs.view(batch_size, *self.bev_shape, -1)
         outputs = outputs.permute(0, 3, 1, 2).contiguous()
         return outputs
+
+
+class ToBEVHeightCompression(nn.Module):
+    def __init__(self,
+                 channels: int,
+                 shape,
+                 offset: list = [0, 0, 0],
+                 dim: int = 1,
+                 bias: bool = False) -> None:
+        super().__init__()
+        self.channels = channels
+        self.offset = torch.cuda.IntTensor([list(offset) + [0]])
+        self.dim = dim
+        self.bev_dims = [i for i in range(3) if i != self.dim]
+        self.bev_shape = shape[self.bev_dims].int()
+        self.shape = shape.int()
+
+    def __repr__(self):
+        return 'ToBEVHeightCompression(channels=%d)' % (self.channels)
+
+    def forward(self, inputs: SparseTensor):
```

Comment:
The full type annotation should also include the return type: `-> torch.Tensor`.